### PR TITLE
feature: Home screen

### PR DESCRIPTION
**Pull request by SergeyMarynenko**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,8 +1,3 @@
 {
-  "presets": ["babel-preset-expo"],
-  "env": {
-    "development": {
-      "plugins": ["transform-react-jsx-source"]
-    }
-  }
-}
+"presets": ["react-native"]
```
Comment (7s4r) [inline .babelrc]: Bad indentation

File: .babelrc [hunk 2]
```diff
@@ -1,8 +1,3 @@
...
+}
```
Comment (7s4r) [inline .babelrc]: EOF

File: .buckconfig [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+[android]
+  target = Google Inc.:Google APIs:23
+
+[maven_repositories]
+  central = https://repo1.maven.org/maven2
```
Comment (7s4r) [inline .buckconfig]: What is this?

File: .directory [hunk 4]
```diff
@@ -1,6 +1,7 @@
 [Dolphin]
-Timestamp=2017,4,20,14,54,8
+Timestamp=2017,4,26,16,56,46
 Version=3
+ViewMode=1
```
Comment (7s4r) [inline .directory]: What is this file?

File: .eslintrc.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
+module.exports = {
+    "parser": "babel-eslint",
```
Comment (7s4r) [inline .eslintrc.js]: bad indentation

File: .eslintrc.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    "extends": "airbnb",
+    "plugins": [
+        "react",
+        "react-native",
+        "jsx-a11y",
+        "import"
+    ],
+    "rules": {
+      "indent": [2, 2],
```
Comment (7s4r) [inline .eslintrc.js]: Why?

File: .eslintrc.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+      "comma-dangle": ["error", "always-multiline"],
+      "semi": [2, "never"],
+      "no-console": ["error", { "allow": ["warn", "error", "info"] }],
+      "max-len": 0,
+      "no-shadow": 0,
+      "no-unused-vars": 0,
+      "no-nested-ternary": 0,
```
Comment (7s4r) [inline .eslintrc.js]: Remove "no-nested-ternary", "no-unused-vars" & "no-shadow"

File: .watchmanconfig [hunk 8]
```diff
@@ -1 +1 @@
-{}
+{}
```
Comment (7s4r) [inline .watchmanconfig]: EOF

File: __tests__/index.android.js [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+import Index from '../index.android.js';
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer';
+
+it('renders correctly', () => {
+  const tree = renderer.create(
+    <Index />
+  );
+});
```
Comment (7s4r) [inline __tests__/index.android.js]: Remove all semicolons

File: __tests__/index.ios.js [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+import Index from '../index.ios.js';
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer';
+
+it('renders correctly', () => {
+  const tree = renderer.create(
+    <Index />
+  );
+});
```
Comment (7s4r) [inline __tests__/index.ios.js]: Remove all semicolons

File: index.android.js [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+import SplashScreen from './src/components/SplashScreen'
+
+const ContainerView = styled.View`
+  flex: 1;
+  background-color: #fff;
+`
+
+const ToolBarView = styled.ToolbarAndroid`
+  background-color: #9b59b6;
+  height: 56;
```
Comment (7s4r) [inline index.android.js]: missing units

File: index.android.js [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+          <ToolBarView
+            title="Garageum"
+            navIcon={require('./public/images/icon_nav.png')}
+            actions={[{
+              title: 'Settings',
+              icon: require('./public/images/icon_settings.png'),
+              show: 'always' }]}
+          />
+        </View>
+         }
```
Comment (7s4r) [inline index.android.js]: Bad indentation

File: index.ios.js [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+    margin: 10,
+  },
+  instructions: {
+    textAlign: 'center',
+    color: '#333333',
+    marginBottom: 5,
+  },
+});
+
+AppRegistry.registerComponent('garageum', () => garageum);
```
Comment (7s4r) [inline index.ios.js]: Remove all semicolons

File: package.json [hunk 14]
```diff
@@ -1,26 +1,36 @@
...
+    "test": "jest"
+  },
+  "dependencies": {
+    "babel-eslint": "^7.2.3",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-1": "^6.24.1",
+    "eslint": "^3.19.0",
+    "eslint-plugin-promise": "^3.5.0",
+    "eslint-plugin-react-native": "^2.3.2",
```
Comment (7s4r) [inline package.json]: put babel & eslint packages to devDependencies

File: src/components/SplashScreen/index.js [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+
+const TitleWrapperView = styled.View`
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+`
+
+const LogoImage = styled.Image`
+  width: 100;
+  height: 100;
```
Comment (7s4r) [inline src/components/SplashScreen/index.js]: missing units

File: src/components/SplashScreen/index.js [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+`
+
+const TitleText = styled.Text`
+  color: #fff;
+  font-size: 35;
```
Comment (7s4r) [inline src/components/SplashScreen/index.js]: missing units

File: src/components/SplashScreen/index.js [hunk 17]
```diff
@@ -0,0 +1,59 @@
...
+`
+
+I18n.translations = {
+  'en-US': {
+    powered: 'Powered by',
+  },
+  fr: {
+    powered: 'Alimenté par',
+  },
+}
```
Comment (7s4r) [inline src/components/SplashScreen/index.js]: Add translations to a dedicated file

File: .buckconfig (hunk 3 above)
Comment (SergeyMarynenko) [inline .buckconfig]: Its for android manifest

**Pull request merged**